stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
```

Comment:
I believe "for the determination" is more grammatically correct, but both sound awkward. "for computing" sounds better.